spec/helpers/recommendations_helper_spec.rb

```diff
@@ -1,4 +1,5 @@
 require "rails_helper"
+require "./db/fixtures/livefrombrexit_recommendations_json"
```

Comment:
I think `require_relative` is the idiomatic way to do this, but this must work so I'm not gonna block merging on a stupidly tiny nitpick ;-)